packages/astro/src/assets/internal.ts

```diff
@@ -110,7 +110,7 @@ export async function getImage(
 		src: imageURL,
 		srcSet: {
 			values: srcSets,
-			attribute: srcSets.map((srcSet) => `${srcSet.url} ${srcSet.descriptor}`).join(', '),
+			attribute: srcSets.map((srcSet) => `${srcSet.url.replaceAll(" ","%20")} ${srcSet.descriptor}`).join(', '),
```

Comment:
should `encodeUri` be used instead?

Reply:
Yeah I think so. I was just thinking that it is a bit specific. I will update momentarily...